config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-addons.yaml

```
@@ -0,0 +1,42 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-no-addon-config-maps-master
```

```suggestion
- name: ci-kubernetes-e2e-kubeadm-kinder-no-addons-master
```
Trying to shorten this up

---

will update.